File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryCabinet.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ContemporaryCabinet implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a contemporary cabinet\n";
    }
}

File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryCabinet.cs
﻿using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ContemporaryCabinet : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a contemporary cabinet");
        }
    }
}
